internal/api/users/list_test.go

```diff
@@ -0,0 +1,72 @@
+package users
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/model"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func TestList(test *testing.T) {
+	usersMap := db.MustGetServerUsers()
+
+	users := make([]*model.ServerUser, 0, len(usersMap))
+	for _, user := range usersMap {
+		users = append(users, user)
+	}
+
+	testCases := []struct {
+		email     string
+		pass      string
```

Comment:
This is a minor change, but since a user's name is always their password,
you can just use `email` in the test case and then lookup the name/password in `usersMap`.